handsontable/src/plugins/customBorders/customBorders.js

```diff
@@ -719,6 +744,27 @@ export class CustomBorders extends BasePlugin {
     }
   }
 
+  /**
+   * Checks the settings cohesion. The properties such like "left"/"right" are supported only
+   * in the LTR mode and the "left"/"right" options can not be used together with "start"/"end" properties.
+   *
+   * @private
+   * @param {object[]} customBorders The user defined custom border objects array.
```

Comment:
According to https://github.com/handsontable/handsontable/pull/9144/files#r793013086

`@param {Array.<BorderDef>}`

Reply:
https://github.com/handsontable/handsontable/pull/9144#discussion_r793326180